### PR TITLE
Register Maxima [https://github.com/nsmith5/Maxima.jl.git]: v0.0.1, v…

### PR DESCRIPTION
**Pull request by nsmith5**
Maxima.jl is a package for symbolic computations in Julia using Maxima CAS

### CODE REVIEW

File: Maxima/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline Maxima/versions/0.0.2/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (nsmith5) [inline Maxima/versions/0.0.2/requires]: 0.4 seems to be it. I'm using Channels which don't appear to be in 0.3

Comment (ararslan) [inline Maxima/versions/0.0.2/requires]: Tony means that you should declare a minimum version of Compat that provides the compatibility features required by your package.

Comment (ararslan) [inline Maxima/versions/0.0.2/requires]: You can update the pull request manually to add the version bound, then update the repo separately. You don't have to close or retag.

* `cd` to your local clone of METADATA, typically in ~/.julia
* Check out a new branch: `git checkout -b maxima-0.0.2`
* Edit Maxima/versions/0.0.2/requires to include the bound on Compat
* Stage and commit the changes (`git add -u`, `git commit -m "Add compat bound"`)
* `git push nsmith5 maxima-0.0.2:pull-request/80ad812e`
* Switch back to your default branch: `git checkout metadata-v2`

Comment (nsmith5) [inline Maxima/versions/0.0.2/requires]: Ah great thanks. Done.

Comment (tkelman) [inline Maxima/versions/0.0.2/requires]: (and be sure to make the corresponding change to https://github.com/nsmith5/Maxima.jl/blob/master/REQUIRE before your next tag)

Comment (nsmith5) [inline Maxima/versions/0.0.2/requires]: Also done :) https://github.com/nsmith5/Maxima.jl/commit/c225e94a0cf684b21a1c483226484aed36aa2e8a

**Comment by ararslan** [comment]
Looks like things are working as expected on Julia 0.5 and 0.6 but a `convert` method is failing on 0.4.

**Comment by nsmith5** [comment]
Supplied missing conversion methods for VERSION < v"0.5.0" https://github.com/nsmith5/Maxima.jl/commit/f1f6e9b2b79d2747ed3caf75ddd4a4e05f64dea4

**Comment by ararslan** [comment]
Okay great. You'll want to do one of two things now:
* Close this, delete the tag, retag with your fix, and resubmit the PR here, _or_
* Add another tag that includes the fix, add the tag to this PR

@tkelman could tell you which is the better option.

**Comment by tkelman** [comment]
Technically the first option doesn't require closing and resubmitting, you could repurpose/modify this same branch. I'd say go with whichever you find easier.

**Comment by nsmith5** [comment]
Ok, I tagged another release and added it to the PR here

File: Maxima/versions/0.0.2/requires (hunk 1 above)
Comment (tkelman) [inline Maxima/versions/0.0.2/requires]: the bound should be added on 0.0.1 as well. that can be done by editing and submitting a PR just changing the metadata requires file.

**Pull request merged**